packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -17,27 +17,65 @@ export const notificationRecipe = recipe({
       gap: 'sm',
       width: '100%',
     }),
+    {
+      backdropFilter: 'blur(12px)',
+    },
   ],
   variants: {
-    intent: semanticColors.reduce(
-      (acc, color) => {
-        acc[color] = [
-          atoms({
-            backgroundColor: `semantic.${color}.default`,
-            borderColor: `semantic.${color}.default`,
-            color: `text.semantic.${color}.default`,
-          }),
-          {
-            vars: {
-              [iconColorVar]:
-                tokens.kda.foundation.color.icon.semantic[color]?.default,
-            },
+    intent: {
+      info: [
+        atoms({
+          backgroundColor: 'semantic.info.subtle',
+          borderColor: `semantic.info.default`,
+          color: `text.semantic.info.default`,
+        }),
+        {
+          vars: {
+            [iconColorVar]:
+              tokens.kda.foundation.color.icon.semantic.info.default,
```

Comment:
I think it's best to stay consistent and use the token function. If we ever change the structure we could fix it in one place.